firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -185,4 +193,12 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
     func done() {
         settingsDelegate?.didFinish()
     }
+
+    // MARK: - ThemeApplicable
+    override func applyTheme() {
+        super.applyTheme()
+        let currentTheme = currentTheme()
+        linkButton.setTitleColor(currentTheme.colors.actionPrimary, for: .normal)
```

Comment:
thanks for the update! do we need this line here? If not, let's remove it so we can rely on the `applyTheme` for the link button.

Reply:
Hi @cyndichin,  we don't need this line here because we don't want to have redundant pieces of code that are doing the same thing, title color is already set in the `LinkButton` class. Thank you 🙏